app/views/devise/registrations/_registration_form.html.erb

```diff
@@ -14,12 +14,17 @@
     </div>
 
     <div class="registration__actions">
+      <% if params[:state] == "new-user" && !Settings::Authentication.allow_email_password_registration && !display_social_login? %>
+        <h2 class="mt-8" style="text-align: center;" aria-live="polite">Sorry to be a bummer...</h2>
+        <p class="mb-8" style="text-align: center;" aria-live="polite">Unfortunately, we do not support creating new accounts right now on our mobile app. If you want create a new account to join <%= community_name %>, please do that on the web.</p>
```

Comment:
Suggestion: I know this wasn't in the RFC, but is there any way we could display the URL of the Forem in question? Unless the user knows to long press on a link to copy the URL, they won't have a way to get the Forem's URL out of this page.